applications/welcome/models/tables.py

```diff
@@ -45,3 +45,60 @@
                 Field('name'),  # Name of variable
                 Field('value', 'text'),  # Json, short please
                 )
+
+#########################################################################
+#These correspond to authentication tables.
+#########################################################################
+
+from gluon.tools import Auth, Service, PluginManager
+
+auth = Auth(db)
+
+## after auth = Auth(db)
+
+db.define_table(
+    auth.settings.table_user_name,
+    Field('first_name', length=128, default=''),
+    Field('last_name', length=128, default=''),
+    Field('email', length=128, default='', unique=True),  # required
+    Field('password', 'password', length=512,  # required
+          readable=False, label='Password'),
+    Field('address'),
+    Field('city'),
+    Field('phone'),
+    Field('registration_key', length=512,  # required
+          writable=False, readable=False, default=''),
+    Field('reset_password_key', length=512,  # required
+          writable=False, readable=False, default=''),
+    Field('registration_id', length=512,  # required
+          writable=False, readable=False, default=''))
+
+custom_auth_table = db[auth.settings.table_user_name]  # get the custom_auth_table
```

Comment:
Why are you using a custom auth table?  Why not simply defining extra fields to the normal auth table?